### PR TITLE
[MDS-652] Adding responsive crumbs, custom divider and icon

### PR DESCRIPTION
**Pull request by tkullisaar**

### CODE REVIEW

File: lib/moon/design/breadcrumb/extended.ex [hunk 1]
```diff
@@ -3,27 +3,80 @@ defmodule Moon.Design.Breadcrumb.Extended do
...
           <li class="flex items-center gap-2 text-trunks">
-            <ArrowsRight class="rtl:rotate-180" :if={index != 0} />
-            <span class={
-              "transition-colors duration-200 hover:text-bulma",
-              "text-bulma font-medium": index == Enum.count(@breadcrumbs) - 1
-            }>
-              <a href={crumb.link} aria-current={index == Enum.count(@breadcrumbs) - 1 && "page"}>{crumb.name}</a>
-            </span>
+            <Icon name={@divider || "arrows_right"} class="rtl:rotate-180" :if={index != 0} />
+            {#if @responsive_crumbs}
```
Comment (phgrey) [inline lib/moon/design/breadcrumb/extended.ex:19]: are these if/else parts so different? can they be collapsed in one?

File: lib/moon/design/breadcrumb/collapsed.ex [hunk 2]
```diff
@@ -3,12 +3,16 @@ defmodule Moon.Design.Breadcrumb.Collapsed do
...
   use Moon.StatefulComponent
 
-  alias Moon.Icons.ArrowsRight
+  alias Moon.Icon
   alias Moon.Design.Button.IconButton
   alias Phoenix.LiveView.JS
 
   prop(breadcrumbs, :list)
 
+  prop(divider, :string)
```
Comment (phgrey) [inline lib/moon/design/breadcrumb/collapsed.ex:12]: watcha thinking about adding @doc attributes in this module?

File: lib/moon_web/pages/design/breadcrumb_page.ex [hunk 3]
```diff
@@ -41,21 +41,12 @@ defmodule MoonWeb.Pages.Design.BreadcrumbPage do
...
-          %{
-            :name => 'breadcrumbs',
-            :type => 'list of Moon.Design.Breadcrumb.Crumb',
-            :required => 'No',
-            :default => '[]',
-            :description => 'Crumbs to display'
-          }
-        ]}
-      />
+      <PropsTable module={Moon.Design.Breadcrumb} />
```
Comment (phgrey) [inline lib/moon_web/pages/design/breadcrumb_page.ex:49]: ... and add all the breadcrumbs's modules here

**Review by phgrey — APPROVED** [review]

File: lib/moon/design/breadcrumb/collapsed.ex (hunk 2 above)
Comment (tkullisaar) [reply lib/moon/design/breadcrumb/collapsed.ex:12]: Will be added later when made headless.

**Pull request merged**